internal/sql/bean/bean.go

```diff
@@ -42,22 +43,68 @@ type ImageScanOutputObject struct {
 
 type Severity int
 
+const (
+	HIGH     string = "high"
+	CRITICAL string = "critical"
+	SAFE     string = "safe"
+	LOW      string = "low"
+	MEDIUM   string = "medium"
+	MODERATE string = "moderate"
+)
+
 const (
 	Low Severity = iota
 	Medium
-	High
 	Critical
+	High
+	Safe
```

Comment:
what is the need of high and safe in devtron severity? these are only to be defined in standard severity. please remove this

Reply:
It is being used in standard severity in ConvertToStandardSeverityUtility(common)